vllm/model_executor/layers/quantization/awq_triton.py

```diff
@@ -0,0 +1,296 @@
+import torch
+import triton
+import triton.language as tl
+
+
+@triton.jit
+def awq_dequantize_kernel(
+        qweight_ptr,  # quantized matrix
+        scales_ptr,  # scales, per group
+        zeros_ptr,  # zeros, per group
+        group_size,  # Should always be 128
```

Comment:
Is there a reason this can't be expanded? 64 is also a popular group size for more accuracy sensitive use cases. In awq_marlin we support `GROUP_SIZES = [-1, 32, 64, 128]`

Reply:
Group sizes [-1, 32, 64, 128] supported

Reply:
awesome!